index.js

```diff
@@ -138,32 +178,38 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
-
+let i = 0
+function takeATicketNumber(line) {
+   i += 1
+   line.push(i)
+   return `Welcome. You are ticket nummber ${i}`
+}
 
 // *** Uncomment the lines below to test
 // console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
+console.log(takeATicketNumber(line))
 // // => `Welcome. You are ticket number 1`
 
-// console.log(takeATicketNumber(line))
+console.log(takeATicketNumber(line))
 // // => `Welcome. You are ticket number 2`
 
-// console.log(nowServing(line))
+console.log(nowServing(line))
 // // => `Currently serving 1.`
 
-// console.log(nowServing(line))
+console.log(nowServing(line))
 // // => `Currently serving 2.`
 
 // console.log(takeATicketNumber(line))
 // // => `Welcome. You are ticket number 3`
 
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red")
 
 // ***** Scope & Closures - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 2", "color: red")
+console.log("%cScope & Closures - Question 2", "color: red")
 
+console.log("Going to finish working on the last one tonight")
```

Comment:
Were you able to solve this?
- Check out [Ian's scrimba on closures](https://scrimba.com/scrim/cLgEmZUZ)
- Or the end of the optional [functions lecture from yesterday](https://youtu.be/RQW3ydDJodA). Jump to `28:30` for the closures section